mne/decoding/transformer.py

```diff
@@ -235,14 +266,18 @@ def inverse_transform(self, X, y=None):
 
         Returns
         -------
-        epochs_data : array, shape (n_epochs, n_channels, n_times)
+        epochs : Instance of epochs
             The original data
         """
         if not isinstance(X, np.ndarray):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(X))
 
-        return X.reshape(-1, self.n_channels, self.n_times)
+        X_orig = X.reshape(-1, self.n_channels, self.n_times)
```

Comment:
infer `n_channels` from a stored `self.info` e.g. `n_chan = len(self.info['chs'])` and `n_times= X.shape[1] / n_chan`